processes/third_party_vuln_submit_form_hacker1.md

```diff
@@ -0,0 +1,59 @@
+> NOTE! Thanks for submitting a report! Please replace *all* the [square] sections below with the pertinent details. Remember, the more detail you provide, the easier it is for us to triage and respond quickly, so be sure to take your time filling out the report!
+
+I would like to report [VULNERABILITY] in [MODULE]
+It allows [DESCRIBE THE IMPACT OF THE VULNERABILITY - E.G READ ARBITRARY FILES, READ DATA FROM DATABASE ETC]
+
+# Module
+
+**module name:** [MODULE NAME]
+**version:** [MODULE VERSION]
+**npm page:** `https://www.npmjs.com/package/[MODULE NAME]`
+
+## Module Description
+
+> Copy description from npm page
+
+## Module Stats
+
+> Replace stats below with numbers from npm’s module page:
+
+Stats
```

Comment:
Upon completion and removal of «> Replace stats…» line, this looks like:

## Module Stats

Stats
1 downloads in the last day
10 downloads in the last week
100 downloads in the last month

~1200 estimated downloads per year

# Vulnerability
…

Reply:
Imo, the «Stats» line is not needed there.

Reply:
agree